pkg/lib/saml/samlbinding/result.go

```diff
@@ -0,0 +1,5 @@
+package samlbinding
+
+type SAMLBindingParseResult interface {
+	SAMLBindingParseResult()
```

Comment:
This method should be private. We do not expect outsider to implement this interface.